src/ring_theory/witt_vector/frobenius.lean

```diff
@@ -282,6 +282,7 @@ lemma coeff_frobenius_char_p (x : 𝕎 R) (n : ℕ) :
   coeff (frobenius x) n = (x.coeff n) ^ p :=
 begin
   rw [coeff_frobenius],
+  letI : algebra (zmod p) R := zmod.algebra _ _,
```

Comment:
I can't put a suggestion here, but this should probably also become an `[algebra (zmod p) R]` parameter instead of a `letI`.

Reply:
Are you suggesting to make these changes now or to wait?

Reply:
Now please :)

Reply:
There are declaration, for example `frobenius_eq_map_frobenius`, that need both `char_p R p` and `algebra (zmod p) R`. I am not sure it is reasonable to have both `[algebra (zmod p) R]` and `[char_p R p]`... what do you think?

Reply:
I suppose there is no instance `algebra (zmod p) R` → `char_p R p` because we'd also need to know `p` is prime? Hmm...

If the `algebra` instance doesn't occur in the statement then it should be fine to create it inline in the proof from a `char_p` passed in.

Reply:
Yes. I propose to leave the witt vectors file as it is. I don't think there is a coherent solution to this `char_p R p` or `algebra (zmod p) R` thing...

Reply:
I don't agree with making this an argument; this is a lemma, so we don't care about the specific algebra data (and it's not used in the statement)

Reply:
I agree, this is the current version, where there is only `char_p R p`.